code/linalg/linalg.c

```diff
@@ -233,6 +234,29 @@ static mp_obj_t linalg_dot(mp_obj_t _m1, mp_obj_t _m2) {
 
 MP_DEFINE_CONST_FUN_OBJ_2(linalg_dot_obj, linalg_dot);
 
+//| def norm(x):
+//|    """
+//|    :param ~ulab.array x: a vector or matrix
+//|
+//|    Computes the norm of a vector or matrix."""
+//|    ...
+//|
+
```

Comment:
Here you could be a bit more specific:

//| def norm(x: ulab.array) -> float:
//|    """
//|    :param ulab.array x
//|
//|	The function takes no parameters, and
//|	returns the 2-norm of a matrix or vector, i.e., sqrt(sum(x*x)), however, without the RAM overhead.
//|	"""
//|

Reply:
'no parameters' ? Or options ?

Reply:
There aren't any, are there? You call `norm(x)`, without `ord`, or `axis`, or anything else.

Reply:
It is the notation, for me x is a parameter, ord, axis, etc, would be options.

Reply:
x would be the argument, I think, but either way is fine with me.

Reply:
Commited.